components/TaskInputForm/index.js

```diff
@@ -114,6 +114,33 @@ export default function Form({ onSubmit, formName, defaultData }) {
     setAddingSubtask(false);
   }, [addingSubtask]);
 
+  useEffect(() => {
+    if (defaultData?.title) {
+      titleInputRef.current.value = defaultData.title;
+    }
+    if (defaultData?.subtasks) {
+      const defaultSubtasks = defaultData.subtasks.map((subtask) => {
+        return { id: uuidv4(), value: subtask };
```

Comment:
A question: Why do you map over your default subtasks and set a new id instead of just formatting the data directly? 🤔 You could for example remove the data from your database and upload an updated data json to change this. If you stay to this approach everytime you get the data you will set a new id for each subtask. Usually we do not want that as it could lead to problems if you use this id for further reference in your data.